extensions/oper-tag.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "`oper-tag` Extension"
+layout: spec
+work-in-progress: true
+copyrights:
+  -
+    name: "David Schultz"
+    period: "2022"
+    email: "me@zpld.me"
+---
+
+## Notes for implementing work-in-progress version
+
+This is a work-in-progress specification.
+
+Software implementing this work-in-progress specification MUST NOT use the
```

Comment:
Given the simplicity of this tag, ircv3/ircv3-ideas#78, and the time spent already in production as solanum.chat/oper, I propose we save everyone time and just call this the `oper` tag.

It can still be refined in future with CAP or tag values.